### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [nuttyartist/notes#433] [Q&A] [answered]
Hi all.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by nuttyartist** [comment, marked as answer]
Yes, it is.

**Reply by ghost** [reply]
Hi nuttyartist.

> Yes, it is.

thank you for feedback.  *Have a great day, and thank you for your patience in answering my question.*

**Reply by bjorn** [reply]
I general, you can find out whether a project is active by looking at [its pulse](https://github.com/nuttyartist/notes/pulse/monthly). :-)
